### PR TITLE
YLP-2633 Customisation of the monitoring alerts from "target & alerts" tab by HCP

### PR DESCRIPTION
**Pull request by TimCanu**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/yourloops/test/integration/assert/monitoring-alerts-parameters.assert.ts [hunk 1]
```diff
@@ -184,6 +184,75 @@ export const checkMonitoringAlertsForPatientSaveButtonMmol = async (): Promise<v
...
+
+  expect(discardButton).toBeEnabled()
+
+  await userEvent.clear(lowBgInput)
+  await userEvent.type(lowBgInput, '3.55')
+  expect(within(monitoringAlertsSection.getByTestId('low-bg-text-field-id')).getByText('Value must be a number')).toBeInTheDocument()
+  expect(discardButton).toBeEnabled()
+  await userEvent.clear(lowBgInput)
+  await userEvent.type(lowBgInput, '4.8')
+  expect(discardButton).toBeEnabled()
```
Comment (celia-rouis) [inline packages/yourloops/test/integration/assert/monitoring-alerts-parameters.assert.ts:216]: **❔ question:** ‏Is it really relevant to check whether the discard button is enabled, given that there is no condition in the code to disable it?

**Review by celia-rouis — APPROVED** [review]

File: packages/yourloops/test/integration/assert/monitoring-alerts-parameters.assert.ts (hunk 1 above)
Comment (TimCanu) [reply packages/yourloops/test/integration/assert/monitoring-alerts-parameters.assert.ts:216]: No, it does not bring any value :+1:

**Pull request merged**